a3p-integration/proposals/z:acceptance/auction.test.js

```diff
@@ -0,0 +1,389 @@
+    },
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 3.448275,
+      },
+    },
+  ],
+};
+
+const pushPricesForAuction = async t => {
+  const oraclesByBrand = new Map();
+  await addPreexistingOracles('ATOM', oraclesByBrand);
+
+  await pushPrices(config.price, 'ATOM', oraclesByBrand, t.context.roundId + 1);
+
+  await retryUntilCondition(
+    () => getPriceQuote('ATOM'),
+    res => res === `+${scale6(config.price).toString()}`,
+    'price not pushed yet',
+    {
+      log: t.log,
+      setTimeout: globalThis.setTimeout,
+      ...t.context.pushPriceRetryOpts,
+    },
+  );
+};
+
+const fundAccts = async (depositorAmt = '100000000', t) => {
+  const retryOpts = t.context.retryOpts.bankSendRetryOpts;
+
+  await bankSend(VALIDATORADDR, GOV1ADDR, `${depositorAmt}${ATOM_DENOM}`),
+    await waitUntilAccountFunded(
+      GOV1ADDR,
+      ambientAuthority,
+      { denom: ATOM_DENOM, value: Number(depositorAmt) },
+      { errorMessage: 'gov1 not funded yet', ...retryOpts },
+    );
+
+  const user1Fund = config.bidsSetup[0].bidderFund;
+  await bankSend(GOV1ADDR, USER1ADDR, `${user1Fund.value}${user1Fund.denom}`);
+  await waitUntilAccountFunded(USER1ADDR, ambientAuthority, user1Fund, {
+    errorMessage: 'user1 not funded yet',
+    ...retryOpts,
+  });
+
+  const gov3Fund = config.bidsSetup[1].bidderFund;
+  await bankSend(GOV1ADDR, GOV3ADDR, `${gov3Fund.value}${gov3Fund.denom}`);
+  await waitUntilAccountFunded(GOV3ADDR, ambientAuthority, gov3Fund, {
+    errorMessage: 'gov3 not funded yet',
+    ...retryOpts,
+  });
+};
+
+const bidByPrice = (price, give, offerId, bidder, t) => {
+  return agopsInter(
+    'bid',
+    'by-price',
+    `--price ${price}`,
+    `--give ${give}`,
+    '--from',
+    bidder,
+    '--keyring-backend test',
+    `--offer-id ${offerId}`,
+  );
+};
+
+const bidByDiscount = (discount, give, offerId, bidder, t) => {
+  return agopsInter(
+    'bid',
+    'by-discount',
+    `--discount ${discount}`,
+    `--give ${give}`,
+    '--from',
+    bidder,
+    '--keyring-backend test',
+    `--offer-id ${offerId}`,
+  );
+};
+
+const placeBids = t => {
+  return [...config.bidsSetup].map(
+    ({ bidder, offerId, price, give, discount }) => {
+      if (price) return bidByPrice(price, give, offerId, bidder, t);
+      return bidByDiscount(discount, give, offerId, bidder, t);
+    },
+  );
+};
+
+const depositCollateral = async t => {
+  const [brandsRaw, retryOptions] = await Promise.all([
+    agoric.follow('-lF', ':published.agoricNames.brand', '-o', 'text'),
+    calculateRetryUntilNextStartTime(),
+  ]);
+  const brands = Object.fromEntries(
+    marshaller.fromCapData(JSON.parse(brandsRaw)),
+  );
+
+  const offerSpec = {
+    id: config.depositor.offerId,
+    invitationSpec: {
+      source: 'agoricContract',
+      instancePath: ['auctioneer'],
+      callPipe: [['makeDepositInvitation']],
+    },
+    proposal: {
+      give: {
+        Collateral: { brand: brands.ATOM, value: 100_000_000n },
```

Comment:
should 100_000_000n go in the config?

Reply:
Oops, you're correct. Thanks...

Reply:
Resolved in https://github.com/Agoric/agoric-sdk/pull/10229/commits/50a0e17318c083daea6fc2628216adf294525e7a